### PR TITLE
Endpoints graph 2

### PR DESCRIPTION
**Pull request by borek2**
Added real data to graph 2. I tried to make the punch card aggregator but because of errors now a PR

### CODE REVIEW

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,51 @@
+0 info it worked if it ends with ok
```
Comment (mboom) [inline npm-debug.log]: Do we need this file in the master branch?

File: src/aggregators/forceLayoutAggregator.js [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    
+    function convertToGraphObject(pullRequests) {
+        var graphObject = {
+            "nodes": [],
+            "links": []
+        };
+        graphObject.nodes.push({
+            "name": "user",
+            "type": "user",
+            "src": "https://avatars2.githubusercontent.com/u/2778466?v=3&s=460"
```
Comment (mboom) [inline src/aggregators/forceLayoutAggregator.js]: Is this testdata?

File: src/aggregators/graph2aggregator.js [hunk 3]
```diff
@@ -8,96 +8,90 @@ function Graph2Aggregator(userName, amountOfPr) {
...
+            var values = pr.sessionLengths;
+            values.unshift(pr.pull_request_number);
+            return values;
         });
-        return objectMatrix;
+        return go;
     }
     
     promise = new RSVP.Promise(function (fulfill) {
         opService
```
Comment (mboom) [inline src/aggregators/graph2aggregator.js]: It looks like this is the old OctopeerService construction. There is no need to contruct an OctopeerService object, you can use the global var octopeerService.

File: src/aggregators/forceLayoutAggregator.js (hunk 2 above)
Comment (borek2) [inline src/aggregators/forceLayoutAggregator.js]: yup, but as i said, i was still working on this, so will do in another pr

File: src/aggregators/graph3aggregator.js [hunk 4]
```diff
@@ -8,18 +8,6 @@ function Graph3Aggregator(userName, amountOfPr) {
     opService = new OctopeerService();
```
Comment (mboom) [inline src/aggregators/graph3aggregator.js]: Also here it looks like this is the old OctopeerService construction. There is no need to contruct an OctopeerService object, you can use the global var octopeerService.

File: src/aggregators/pullRequestsAggregator.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+                dictionary[session.pull_request.url] = counter;
+                pullRequests.push(session.pull_request);
+                counter += 1;
+            }
+        });
+        return pullRequests;
+    }
+    
+    promise = new RSVP.Promise(function (fulfill) {
+        octopeerService
```
Comment (mboom) [inline src/aggregators/pullRequestsAggregator.js]: Good job here! This is the new octopeerService construction.
